airflow/www/views.py

```diff
@@ -4514,6 +4514,13 @@ def _clean_description(self, description):
         def _build_link(match_obj):
             text = match_obj.group(1)
             url = match_obj.group(2)
+
+            # parsing the url to check if ita a valid url
+            parsed_url = urlparse(url)
+            if not (parsed_url.scheme == "http" or parsed_url.scheme == "https"):
+                # returning the original raw text
+                return escape(match_obj.group(0))
+
```

Comment:
Could you add a test for this change?
It is not just a validation, where the returned value would be different in some cases.

Reply:
Yeah, valid ask. I just pushed test cases.